commercial/app/views/static/adFreeSurveyPage.scala.html

```diff
@@ -11,18 +11,10 @@
         @fragments.metaData(adFreeSurvey)
 
         @* get the stylesheets downloading ASAP *@
-        @fragments.stylesheetLinks(None)
+        @fragments.stylesheets(None)
 
-        @* polyfill, feature detect etc before we try and use the stylesheets *@
-        @fragments.javaScriptPreFlight(adFreeSurvey)
-
-        @* try and load fonts before we use the stylesheets *@
-        @fragments.fonts()
-
-        @* start trying to use the stylesheets *@
-        @fragments.stylesheetLinksEnable()
-
-        @fragments.javaScriptBootstraps(adFreeSurvey)
+        @* inline JS - blocking *@
+        @fragments.inlineJSBlocking(adFreeSurvey)
```

Comment:
Hey @sndrs. Did you test this page after making the change? It might have the same problems I had on the offline page, just FYI.